morituri/command/cd.py

```diff
@@ -279,15 +295,20 @@ def add_arguments(self):
         self.parser.add_argument('--track-template',
             action="store", dest="track_template",
             default=DEFAULT_TRACK_TEMPLATE,
-            help="template for track file naming (default default)")
+            help="template for track file naming")
```

Comment:
these (and the above whitespace change in getPath) are both good changes, but I think don't belong in this PR.

Reply:
The whitespace change seems to have been included already by some other change. I'll take out the default-default patch out of it and make a new pull request in a moment.

Reply:
PR : https://github.com/JoeLametta/whipper/pull/188